### PR TITLE
Added average metric AVG() to default metrics

### PR DESCRIPTION
**Pull request by vera-liu**
Issue: https://github.com/airbnb/caravel/issues/804

Done: Added avg to default metrics in models->SqlaTable

screen shot 2016-10-25 at 11 29 16 am
screen shot 2016-10-25 at 11 29 31 am

needs-review @bkyryliuk @mistercrunch

### CODE REVIEW

File: caravel/models.py [hunk 1]
```diff
@@ -1101,6 +1101,13 @@ def fetch_metadata(self):
                     metric_type='sum',
                     expression="SUM({})".format(quoted)
                 ))
+            if dbcol.sum:
```
Comment (bkyryliuk) [inline caravel/models.py]: s/dbcol.sum/dbcol.avg ?

Comment (bkyryliuk) [inline caravel/models.py]: it would be great to have a unit test here

Comment (vera-liu) [inline caravel/models.py]: Do we have tests covering metrics/groupby of db queries? I was trying to look for them but didn't seem to exist core_tests.py

Comment (bkyryliuk) [inline caravel/models.py]: nope, you'll be a first one if you'll decide to do it

Comment (bkyryliuk) [inline caravel/models.py]: dbcol.max and dbcol.min are also in similar relation.
@mistercrunch  - should avg be a separate field in the TableColumn ?

Comment (vera-liu) [inline caravel/models.py]: Since it will be tests covering all metrics, I'll separate it to another PR ;)

Comment (bkyryliuk) [inline caravel/models.py]: I've talked to Max, please implement a separate checkbox for AVG metric.
User may not expect to add 2 metrics when setting the `sum` checkbox and will be tricky to explain him if you want to add `avg` metrics - please select sum.

**Comment by vera-liu** [comment]
Added a new commit, making avg a column for SqlaTable and Druid class @bkyryliuk

**Comment by bkyryliuk** [comment]
please fix the tests

**Comment by vera-liu** [comment]
Added migration 😃  @bkyryliuk

File: caravel/migrations/versions/ad4d656d92bc_add_avg_metric.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# revision identifiers, used by Alembic.
+revision = 'ad4d656d92bc'
+down_revision = '7e3ddad2a00b'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('columns', sa.Column('avg', sa.Boolean(), nullable=True))
```
Comment (bkyryliuk) [inline caravel/migrations/versions/ad4d656d92bc_add_avg_metric.py]: do the same for the upgrade

**Comment by bkyryliuk** [comment]
LGTM

**Comment by mohadelrezk** [comment]
Hi, 
I am using latest version of superset (0.24)
but average is not activated as a metric

**Pull request merged**